.circleci/config.yml

```
@@ -268,7 +262,7 @@ jobs:
             cd metacoin/
             # `truffle test` compiles the project but artifacts go into /tmp/
             ! [[ -e build/ ]] || false
-            echo "module.exports['compilers'] = {solc: {version: '$(realpath ../solc-js/)'}}" > truffle-config.js
+            echo "module.exports['compilers'] = {solc: {version: '$(realpath ../solc-js/dist)'}}" > truffle-config.js
```

I think it would probably be best to change it to:
```suggestion
            echo "module.exports['compilers'] = {solc: {version: '$(realpath node_modules/solc/)'}}" > truffle-config.js
```

This way again we don't have to make an independent decision to use `dist/` but just use whatever gets published.

I think I'll change this in my PR directly.

---

Yeap all these changes should happen in your PR when you introduce the changes.
I just make the change to fix the test and did't know of the correct location it should happen.

---

I mean, your change is good and in the correct place. I'm just saying that there's a more general solution and I should have used it.

And yeah, these changes should be in my PR but it makes sense to put them there only after it's on top of your PR, i.e. after yours gets merged.